File: 455-AssignCookies/455-AssignCookies.cpp
// Last updated: 07/04/2026, 11:38:53
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        int n = g.size(),m = s.size(),l=0,r=0;
        sort(g.begin(),g.end());
        sort(s.begin(),s.end());
        while(l<n && r<m){
            if(s[r]>=g[l])
                l++;
            r++;
        }
        return l;
    }
};

File: 455-AssignCookies/455-AssignCookies.py
# Last updated: 07/04/2026, 11:38:57
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        s.sort()
        g.sort()
        c = 0 
        S = G = 0 
        while S <len(s)  and G < len(g) :
            if s[S] >= g[G] :
                c+=1
                S+=1
                G+=1
            else:
                S+=1

        return c




       